python/test/unit/language/test_core.py

```diff
@@ -3367,48 +3367,55 @@ def kernel(X, stride_xm, stride_xk, Y, stride_yk, stride_yn, W, stride_wn, strid
             assert 'wgmma.mma_async.sync.aligned.m64n128k32.f32.e4m3.e4m3' in ptx
 
 
-@pytest.mark.parametrize("M, N, K, col_a, col_b, type_a, type_b, num_warps, mma, kpack",
-                         [(M, N, K, col_a, col_b, type_a, type_b, 4, mma, kpack)
+@pytest.mark.parametrize("M, N, K, col_a, col_b, rhs_scale, elemtent_type, scale_type, num_warps, mma, kpack",
```

Comment:
typo: element

FWIW, I find the `element_type` and `scale_type` quite confusing here--`scale_type` is actually for the operand that does not need to be scaled IIUC. What about `mxfp_type` and `normal_type`?